### PR TITLE
feat : completed landing page for my e-commerce Blossom store.

### PR DESCRIPTION
**Pull request by shuklahimanshu81**
Completed home page for my e-commerce Blossom store.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+body{
+    background-color: var(--background-color);
+}
+nav{
+    height: 5rem;
+    background-color: var(--background-color);
+}
+ion-icon{
+    color: black;
```
Comment (stutisk) [inline index.css:9]: Use variables

**Review by stutisk — COMMENTED** [review]
Other then this code looks great to be merged

File: index.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    border: 2.5px solid var(--black);
+}
+.main-section{
+    width: 90vw;
+    margin: 2.5vh 5vw;
+    flex-direction: column;
+    gap: 1rem;
+    justify-content: center;
+    align-items: center;
+
```
Comment (ZuberDunge) [inline index.css:21]: avoid extra spacing.

File: index.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+}
+.featured-product-section{
+    width: inherit;
+    display: flex;
+    gap:1.5rem;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+}
```
Comment (ZuberDunge) [inline index.css:30]: add spacing after each {}

**Review by ZuberDunge — APPROVED** [review]
Added few suggestions, apart from that everything looks fine. Good to go from my side

File: index.css (hunk 1 above)
Comment (vidhi-thakur) [reply index.css:9]: Also, please try using class selectors

**Review by vidhi-thakur — APPROVED** [review]

File: index.css (hunk 3 above)
Comment (shuklahimanshu81) [reply index.css:30]: Will keep that in mind zuber, thanks.

**Pull request merged**